packages/block-editor/src/components/text-transform-control/stories/index.story.js

```diff
@@ -1,33 +1,74 @@
+/**
+ * Internal dependencies
+ */
+import TextTransformControl from '../';
+
 /**
  * WordPress dependencies
  */
 import { useState } from '@wordpress/element';
 
 /**
- * Internal dependencies
+ * Meta configuration for Storybook
  */
-import TextTransformControl from '../';
-
-export default {
+const meta = {
 	title: 'BlockEditor/TextTransformControl',
 	component: TextTransformControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'Control to facilitate text transformation selections.',
+			},
+		},
+	},
 	argTypes: {
-		onChange: { action: 'onChange' },
+		onChange: {
+			action: 'onChange',
+			control: {
+				type: null,
+			},
+			description: 'Callback function when text transform changes',
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+		},
+		className: {
+			control: { type: 'text' },
+			description: 'Additional CSS class name to apply',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		value: {
+			control: { type: null },
+			description: 'Currently selected writing mode.',
+			table: { type: { summary: 'string' } },
+		},
```

Comment:
As with [the `TextAlignmentControl` component](https://github.com/WordPress/gutenberg/blob/6e8588865ed61c990227dfb9d1324a52f4ff39af/packages/block-editor/src/components/text-alignment-control/stories/index.story.js#L42-L49), it may be a good idea to make the `options` prop controllable by adding the `options` field.